src/main/resources/openapi/transcriptions.yaml

```diff
@@ -112,6 +112,62 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/TranscriptionRequestDetails'
+
+    patch:
+      tags:
+        - Transcription
+      summary: Patch a transcription
```

Comment:
I guess this summary should probably be something like `Update multiple transcriptions to hide request from requestor`?

Reply:
as this is a generic method, that may update more in the future, i have renamed to `Update multiple transcriptions`